src/DotNetWorker.Core/DotNetWorker.Core.csproj

```diff
@@ -22,6 +22,16 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Core" Version="1.37.0" />
```

Comment:
Update Azure.Core to the latest version

Reply:
This package version update is unrelated to the changes we are adding in this PR (net9 TFM support). We can upgrade it in the 2x branch.